source/Calamari.Tests/Fixtures/PackageDownload/PackageDownloadFixture.cs

```diff
@@ -68,6 +70,21 @@ public void ShouldDownloadPackage()
         }
 
         [Test]
+        public void ShouldDownloadPackageWithRepositoryMetadata()
+        {
+            var result = DownloadPackage(NuGetFeed.PackageId, NuGetFeed.Version, NuGetFeed.Id, NuGetFeedUri);
+
+            result.AssertSuccess();
+
+            result.AssertOutput(
+                $"Downloading NuGet package {NuGetFeed.PackageId} {NuGetFeed.Version} from feed: '{NuGetFeedUri}'");
+            result.AssertOutput($"Downloaded package will be stored in: '{NuGetFeed.DownloadFolder}");
+
+            AssertStagePackageOutputVariableSet(result, NuGetFeed.File);
+            result.AssertOutput($"Package {NuGetFeed.PackageId} {NuGetFeed.Version} successfully downloaded from feed: '{NuGetFeedUri}'");
+        }
```

Comment:
Do we need a similar test on the server?

Reply:
That'd probably be a good idea, I'll add it before merging the server PR